site/public/css/officeHoursQueue.css

```
@@ -20,8 +20,11 @@ th {
     display: none;
 }
 
-.current_queue_row,
-a > i.fas {
+.current_queue_row {
+    color: var(--btn-text-black);
+}
+
+.current_queue_row a > i.fas {
     color: var(--btn-text-black);
 }
```

Since we're showing the exact color in both cases, we can condense this to one definition:

```suggestion
.current_queue_row,
.current_queue_row a > i.fas {
    color: var(--btn-text-black);
}
```